<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionGrantXP</class>
 <widget class="QWidget" name="ActionGrantXP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblTargetType">
       <property name="text">
        <string>Target Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="targetType">
       <item>
        <property name="text">
         <string>Character</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Partner</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Character &amp; Partner</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblXP">
       <property name="text">
        <string>XP:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="xp">
       <property name="minimum">
        <number>-2147483647</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblAdjustable">
       <property name="text">
        <string>Adjustable:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="adjustable"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
